docs/versions/latest/Release.md

```diff
@@ -36,6 +36,17 @@
 [//]: # (Optional But higlhy recommended Specify *NONE* if missing)
 [//]: # (#### Relevant Documentation:)
 
+### Added vRLI alert API fallback optional
+
+#### Previous Behaviour
+
+When pushing vROPs enabled vRLI alerts to vRLI 8.8 and later it uses always the v2 alerts API, that causes some vROPs enabled vRLI alerts not to be pushed
+despite the pushing would work fine with the v1 alerts API.
+
+#### New Behaviour
+An option flag has been added (useOldAlertsApi / vrli_use_old_alerts_api) so that customer is able to control which alerts API to be used. By default the flag
```

Comment:
Is this the best way to handle this? Would it perhaps make more sense to fallback to v1 in case of v2 failure ( one that we expect )

I am just thinking in the long run, this isn't going to work, as clients will be forced to use v1 to workaround the bug, but will be unable to utilize v2's new features

Reply:
Actually currently this is the best case to handle this, as the alerts v2 API has some bugs. This is an optional flag for handling some corner cases of importing vROPs enabled vRLI alerts.